README.md

```diff
@@ -1,18 +1,27 @@
 # Hello-Microverse
 
-> One paragraph statement about the project.
-> Description the project.
+> One h1 Tittle about the project.
+> Create branches with different base branches.
+> Understand the concept of version control.
+> Follow GitHub flow.
+> Open pull requests in GitHub.
+> Write descriptive, easy to understand commit messages.
+> Write short, detailed, easy to understand descriptions of pull requests.
+> Write a descriptive and easy to understand README in business English for every project submitted.
+> Demonstrate an ability to submit a project for code review..
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+# Descrption
 
-## Built With
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
 
-- Major languages
-- Frameworks
-- Technologies used
+## General requirements
 
-## Live Demo
-## Live Demo (if available)
+- Make sure that there are no linter errors.
+- Make sure that you used correct (GitHub flow).
+- Make sure that you documented your work.
+
+## Waiting for the review code.
+## I hope i have luck (if that is ok)
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] Kindly make sure you use the [Microverse README template](https://github.com/microverseinc/readme-template) as required for the project and correctly complete sections related to your project(e.g.: **Project title as you did, good project description as you did, Live Demo section (if available), Getting Started section, Authors section, Contributing section, Show your support section, Acknowledgments section, License section**,... ) then remove the non-related sections as required for the project. You can look at [this template for guidelines](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md).